### ISSUE TITLE
Add inspect on exception or wait on exception command line option

### ISSUE THREAD

**Issue by ironpythonbot** [IronLanguages/main#927]
Python has a -i option which always goes into inspect mode regardless of how
the process exited. It'd be nice to have a -X:InspectOnException or
-X:WaitOnException option to leave a console window up when an exception has
been unhandled. This would be useful for windows apps which launch seperate
console windows.  
## Work Item Details

**Original CodePlex Issue:**    [Issue 30278](http://ironpython.codeplex.com/workitem/30278)
**Status:** Proposed
**Reason Closed:**  Unassigned
**Assigned to:**    Unassigned
**Reported on:**    Mar 2, 2011 at 9:12 PM
**Reported by:**    DinoViehland
**Updated on:** Feb 22, 2013 at 2:09 AM
**Updated by:** jdhardy
